### ISSUE TITLE
Cannot find module './ast.core/linter'

### ISSUE THREAD

**Issue by splitice** [raml-org/raml-js-parser-2#889]
Current release is broken.

```
Cannot find module './ast.core/linter'
```

```
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/circleci/repo/dist/node_modules/raml-1-parser/dist/parser/lowLevelAST.js:6:14)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/circleci/repo/dist/node_modules/raml-1-parser/dist/parser/jsyaml/jsyaml2lowLevel.js:4:16)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
```

**Comment by postatum** [comment]
Thank you for the report. Investigating.

**Comment by postatum** [comment]
Fixed in version `1.1.55` by https://github.com/raml-org/raml-js-parser-2/commit/92943126d3b939ace9a39ec5779a9977333bddfb

Happened because of this change in npm https://github.com/npm/npm-packlist/commit/800f03dcf2dec360050c57fa665925182e88e6b7 which was partially fixed by this https://github.com/npm/npm-packlist/commit/9230bb88bfafac94e828cdc5cf3cb29a0248f85f but in our case `ast.core` was still excluded from npm package.

Thanks again for the report @splitice :apple: !
